cmake/FindTensorFlow.cmake

```diff
@@ -0,0 +1,29 @@
+###############################################################################
+# - Find TensorFlow
```

Comment:
Does TF have its own cmake module file?

Reply:
> Does TF have its own cmake module file?

I didn't find it, maybe we could use this file temporarily..